prost-build/src/lib.rs

```diff
@@ -149,7 +149,7 @@ mod message_graph;
 mod path;
 
 mod config;
-pub use config::Config;
+pub use config::{protoc_from_env, protoc_include_from_env, Config};
```

Comment:
We should restore all move `pub` functions. This version will satisfy cargo-semver-checks:

```suggestion
pub use config::{error_message_protoc_not_found, protoc_from_env, protoc_include_from_env, Config};
```

Reply:
> Sorry, I accepted the change that broke your build. I was naive of me to assume this is an “internal” function and nobody is using it.

No worries. It was great to see it quickly yanked and sorted :raised_hands:. Thanks for spending your time working on the crates I use :)

> We should restore all moved pub functions. This version will satisfy cargo-semver-checks:

:+1: https://github.com/tokio-rs/prost/compare/1e5e904b47773b92ea46ba6e82feaaa2f006a9f4..f4be7b750a6039c8535ae3e105cf912c3b0c12e7

Reply:
> No worries. It was great to see it quickly yanked and sorted 🙌. Thanks for spending your time working on the crates I use :)

Well, you created this PR half an hour after publish. So, I am impressed with that as well :smiley: